lib/getImageExtensionFromMime.ts

```diff
@@ -0,0 +1,14 @@
+const extensionsMap: Record<string, string> = {
+	'image/apng': 'apng',
+	'image/avif': 'avif',
+	'image/gif': 'gif',
+	'image/jpeg': 'jpg',
+	'image/png': 'png',
+	'image/svg+xml': 'svg',
+	'image/webp': 'webp',
+	'image/bmp': 'bmp',
+	'image/x-icon': 'ico',
+	'image/tiff': 'tif',
```

Comment:
I'd remove these as they support on every browser is not guaranteed.

Reply:
As we discussed privately, I'll keep these since they're not related to the image previews.